### PR TITLE
update governance - Add new SC positions, AGM, DEI mission

### PR DESCRIPTION
**Pull request by cmaimone**

### CODE REVIEW

**Review by danielskatz — APPROVED** [review]

**Review by JeffCarver — APPROVED** [review]

**Review by jdamerow — APPROVED** [review]

**Review by nicole-brewer — APPROVED** [review]

File: governance.md [hunk 1]
```diff
@@ -11,15 +11,14 @@ strives to:
 
 * provide a coherent professional community in the United States for those who identify as a
     Research Software Engineer (RSE),
-* advocate for RSE professionals, and
-* provide resources to support and advance RSEs and the RSE career path within the United States
-    research ecosystem.
+* advocate for RSE professionals, 
+* provide resources to support and advance RSEs and the RSE career path within the United States research ecosystem, and
```
Comment (lparsons) [inline governance.md]: I don't think the "and" is needed
```suggestion
* provide resources to support and advance RSEs and the RSE career path within the United States research ecosystem
```

File: governance.md [hunk 2]
```diff
@@ -90,20 +91,31 @@ available.
...
 the Steering Committee.
 
 #### Chair
 
-The Chair will set the agenda and preside over Steering Committee meetings or arrange for the Vice-
-Chair or another Steering Committee member to do so.
+The Chair will set the agenda and preside over Steering Committee meetings or arrange for the Vice 
+Chair or another Steering Committee member to do so.  
+The Chair is responsible for convening the Annual General Meeting of the membership.  
+The Chair is responsible for reporting on US-RSE activities over the past year at the Annual General Meeting.
```
Comment (lparsons) [inline governance.md:113]: This seems like a duplicate of the Vice Chair duty to "annually prepare a report of Association activities in the past year".

**Review by lparsons — CHANGES_REQUESTED** [review]
I had three suggestions for changes before ratifying this:

1. Small grammatical change on line 15

2. Clarification of annual reporting duties (Chair or Vice Chair)

3. Specific DEI related wording:

Under "Membership", I suggest we align the wording between this document, the DEI Mission State, and the Code of Conduct.

I suggest we change this document to match the specific language in the DEI Mission Statement:
> As stated in our Code of Conduct, we welcome and respect individuals of all dimensions of diversity, including but not limited to race, color, caste, economic status, gender expression, gender identity, sexual orientation, disability, neurocognitive differences, age, religion (or lack thereof), national origin, and ethnicity.

We can make a separate PR to update the Code of Conduct to match once this is accepted.

File: governance.md (hunk 2 above)
Comment (cmaimone) [reply governance.md:113]: Can you suggest a specific change you want to make so we can just commit it?

**Comment by cmaimone** [comment]
@lparsons same with the DEI comment - can you propose any specific changes needed to this document here so we can easily incorporate it?

**Review by cosden — APPROVED** [review]

**Review by lparsons — APPROVED** [review]

**Review by christophernhill — APPROVED** [review]
All looks good - Chris

**Review by sandragesing — APPROVED** [review]

**Pull request merged**